packages/hash/external-services/kratos/kratos.yml

```diff
@@ -0,0 +1,61 @@
+# Based on the quick-start example https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml
+
+version: v0.7.1-alpha.1
+
+dsn: memory
+
+serve:
+  public:
+    base_url: http://localhost:4433/
+    cors:
+      enabled: true
+  admin:
+    base_url: http://kratos:4434/
+
+selfservice:
+  default_browser_return_url: http://localhost:3000/
+  allowed_return_urls:
+    - http://localhost:3000
+
+  methods:
+    password:
+      enabled: true
+
+  flows:
+    error:
+      ui_url: http://localhost:3000/error
+
+    logout:
+      after:
+        default_browser_return_url: http://localhost:3000/login
+
+    login:
+      ui_url: http://localhost:3000/login
+      lifespan: 10m
+
+    registration:
+      lifespan: 10m
+      ui_url: http://localhost:3000/signup
+      after:
+        password:
+          hooks:
+            - hook: session
+
+log:
+  level: debug
+  format: text
+  leak_sensitive_values: true
```

Comment:
Do we want to do this

Reply:
I've been treating this directory as the developer configuration for kratos, as the production configuration will likely quite different. For that use-case I've found having all the debug logs useful.

Do you want me to make this more clear somehow? Perhaps a comment in the file, or rename it to `kratos.dev.yml`?

Reply:
I guess it depends on how we want to do the folder structure for the environments. If we're likely to just name the yamls then I'd rename this as you suggest :)